packages/muelu/src/Interface/MueLu_FactoryFactory_decl.hpp

```diff
@@ -197,52 +199,54 @@ namespace MueLu {
+      if (factoryName == "PgPFactory")                         return Build2<PgPFactory>                         (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "SaPFactory")                         return Build2<SaPFactory>                         (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "RAPFactory")                         return BuildRAPFactory<RAPFactory>                (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "RebalanceAcFactory")                 return Build2<RebalanceAcFactory>                 (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "RebalanceTransferFactory")           return Build2<RebalanceTransferFactory>           (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "ReorderBlockAFactory")               return Build2<ReorderBlockAFactory>               (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "SegregatedAFactory")                 return Build2<SegregatedAFactory>                 (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "SubBlockAFactory")                   return Build2<SubBlockAFactory>                   (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "TentativePFactory")                  return Build2<TentativePFactory>                  (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "ToggleCoordinatesTransferFactory")   return BuildToggleCoordinatesTransferFactory      (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "TogglePFactory")                     return BuildTogglePFactory<TogglePFactory>        (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "TransPFactory")                      return Build2<TransPFactory>                      (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "TrilinosSmoother")                   return BuildTrilinosSmoother                      (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "UncoupledAggregationFactory")        return Build2<UncoupledAggregationFactory>        (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "UserAggregationFactory")             return Build2<UserAggregationFactory>             (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "UserPFactory")                       return Build2<UserPFactory>                       (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "SemiCoarsenPFactory")                return Build2<SemiCoarsenPFactory>                (paramList, factoryMapIn, factoryManagersIn);
+      if (factoryName == "GeometricCoordinatesTransferFactory")return Build2<GeometricCoordinatesTransferFactory>(paramList, factoryMapIn, factoryManagersIn);
```

Comment:
Sure, should I also clean-up the entries that where put there before mine to have the list fully alphabetical or is there a reason to keep "SemiCoarsenPFactory" and "RepartitionInterface" at the bottom of the list?